app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Interfaces/APICallerInterface.java

```diff
@@ -0,0 +1,20 @@
+package edu.grinnell.appdev.grinnelldirectory.Interfaces;
+
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+
+/**
+ * Created by nicholasroberson on 2/15/17.
```

Comment:
Class name is pretty obvious but what if we have more APIs in the app? Documentation will be useful here.